Portals/_default/Skins/Vanjaro/Base.ascx.cs

```diff
@@ -17,6 +18,7 @@
 using System.Dynamic;
 using System.IO;
 using System.Linq;
+using System.Reflection;
```

Comment:
Why do we need reflection?

Reply:
Its added by mistake and removed  from the code